src/robusta/core/sinks/robusta/robusta_sink.py

```diff
@@ -309,6 +315,45 @@ def __publish_new_jobs(self, active_jobs: List[JobInfo]):
         self.dal.publish_jobs(updated_jobs)
 
     def __update_cluster_status(self):
+        global_config = self.get_global_config()
+        activity_stats = ActivityStats(
+            relayConnection=False,
+            alertManagerConnection=False,
+            prometheusConnection=False,
+            prometheusRetentionTime='',
+        )
+
+        # checking the status of relay connection
+        receiver = self.registry.get_receiver()
+        if isinstance(receiver, ActionRequestReceiver):
+            activity_stats.relayConnection = receiver.healthy
+
+        # checking the status of prometheus
+        try:
+            prometheus_params = PrometheusParams(prometheus_url=global_config.get("prometheus_url", ""))
+            prometheus_connection = get_prometheus_connect(prometheus_params=prometheus_params)
+            check_prometheus_connection(prom=prometheus_connection, params={
```

Comment:
This query should be used if we want to see if some data returned.
If we don't, there's no point sending a custom query (the default query in `check_prometheus_connection` is enough)

Reply:
done